src/app/layout.tsx

```diff
@@ -4,8 +4,19 @@ import { Toaster } from '@/components/ui/toaster';
 import './globals.css';
 
 export const metadata = {
```

Comment:
Nice work with this!! I think the current logo we're using is `/connectient-logo.png`